src/profile

```diff
@@ -45,17 +45,21 @@ function _glibc_check () {
 _glibc_check
 
 _crew_bash () {
-# Bash in path will be the Chromebrew bash if it is installed.
-# Switch to Chromebrew bash as early as possible if it is installed.
-if [ -x "$CREW_PREFIX/bin/bash" ]; then
-  CREW_BASH_VERSION=$(LD_LIBRARY_PATH="$CREW_LIB_PREFIX":/lib$LIB_SUFFIX:/usr/lib$LIB_SUFFIX "$CREW_PREFIX"/bin/bash --norc --version | LD_LIBRARY_PATH= /usr/bin/awk 'NR==1{print $4}')
-  if [ -n "$CREW_BASH_VERSION" ] && [ "${CREW_BASH_VERSION}" != "${BASH_VERSION}" ]; then
-    echo "Starting Chromebrew bash."
-    exec env LD_LIBRARY_PATH="$CREW_LIB_PREFIX":/lib$LIB_SUFFIX:/usr/lib$LIB_SUFFIX "$CREW_PREFIX"/bin/bash "$@"
+  # Bash in path will be the Chromebrew bash if it is installed.
+  # Switch to Chromebrew bash as early as possible if it is installed.
+  if [ -x "$CREW_PREFIX/bin/bash" ]; then
+    CREW_BASH_VERSION=$(LD_LIBRARY_PATH="$CREW_LIB_PREFIX":/lib$LIB_SUFFIX:/usr/lib$LIB_SUFFIX "$CREW_PREFIX"/bin/bash --norc --version | LD_LIBRARY_PATH= /usr/bin/awk 'NR==1{print $4}')
+    if [ -n "$CREW_BASH_VERSION" ] && [ "${CREW_BASH_VERSION}" != "${BASH_VERSION}" ]; then
```

Comment:
Where is `BASH_VERSION` set or is this already an available environment variable?

Reply:
BASH_VERSION is internal to bash. it is set automatically.